pages/admin/AdminFunctionsHandler.inc.php

```diff
@@ -109,6 +109,31 @@ function clearDataCache() {
 
 		Request::redirect(null, 'admin');
 	}
+
+	/**
+	 * Download scheduled task execution log file.
+	 * @param $args array
+	 */
+	function downloadScheduledTaskLogFile() {
+		$this->validate();
+		$application =& Application::getApplication();
+		$request =& $application->getRequest();
+
+		$file = $request->getUserVar('file');
```

Comment:
actually I dreamed that I did enforced that the file should be inside the scheduled tasks folder. One can pass something like '../../anything here' and access the other files. You're right, I will fix this.